biopet-autodoc/templateparser.py

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env python2
+
+#Script to document new tools using a jinja2 template
+from jinja2 import Environment, FileSystemLoader
+import os
+import argparse
+import ast
+import yaml
+
+if __name__ == "__main__":
+    
+    #Block with command line options
+    parser=argparse.ArgumentParser(description="")
```

Comment:
The `description` is an optional variable. If you set it to an empty string, you may as well leave it out altogether ;-).